tests/unit/BusinessTable.spec.ts

```diff
@@ -45,12 +45,16 @@ describe('Business Table - display', () => {
       type: AmlTypes.LEAR,
       identifier: 'BC1111111',
       name: 'My BC Limited Company',
-      email: 'bc1111111@example.com',
-      address: {
-        streetAddress: '123 Main St',
-        addressCity: 'Victoria',
-        addressCountry: 'CA',
-        postalCode: 'V8V 8V8'
+      authInfo: { contacts: [ { email: 'bc1111111@example.com' } ] },
+      addresses: {
+        registeredOffice: {
+          mailingAddress: {
+            streetAddress: '123 Main St',
+            addressCity: 'Victoria',
+            addressCountry: 'CA',
+            postalCode: 'V8V 8V8'
+          }
+        }
```

Comment:
Yesterday, I changed the code to save the entire office addresses object instead of just the registered mailing address.